src/main/java/randoop/execution/ProcessException.java

```diff
@@ -0,0 +1,16 @@
+package randoop.execution;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * Exception representing an error that occurs while running a process with {@link
+ * RunEnvironment#run(List, File, long)}.
+ */
+public class ProcessException extends Throwable {
+  private static final long serialVersionUID = 736230736083495268L;
+
+  public ProcessException(String s, Throwable e) {
```

Comment:
The Java 8 javadoc doesn't have this flag, and I'm not seeing it any documentation. 

There is the accepted answer to this question on SO:
https://stackoverflow.com/questions/29519085/how-to-fail-gradle-build-on-javadoc-warnings
but it seems potentially fragile